linebot/flex.go

```diff
@@ -17,7 +17,15 @@ package linebot
 import "encoding/json"
 
 // FlexContainerType type
-type FlexContainerType string
+type FlexContainerType = string // alias type
+
+// FlexValueType type
+type FlexValueType = int // alias type
+
+// FlexInt is a helper function for using *FlexValueType values
+func FlexInt(v int) *FlexValueType {
```

Comment:
FlexInt is not necessary when using type alias.

Reply:
hmm.., how pass `0` as value?
https://github.com/line/line-bot-sdk-go/pull/129/files#diff-40bd644208f7397ac51144bbe03b721bR387

Reply:
I think `*FlexValueType` is not necessary. I wonder why we have to embedded the `*` (reference) instead of `value` 🤔

Reply:
Did you read #124 ?
`flex` is an optional field and tagged as `omitempty`, but it should accept the value `0` as not the default value.
So I thought that we have to embed the `*` (reference) instead of `value` to omit this field only if the reference is null.

Reply:
👍 @sugyan Sound reasonable to me. Thank you very much for explanation.